source/android/uitestapp/src/AdaptiveCardObjectModel.i

```diff
@@ -0,0 +1,217 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
```

Comment:
Is there no way to consume the other `.i` file here? it'd be a bummer if we have to keep them both up to date...

Reply:
+1, the object model should be included by depending on the `adaptivecards` module, right?

Reply:
+1, the object model should be included by depending on the adaptivecards module, right?